packages/mobile/src/account/Settings.tsx

```diff
@@ -320,6 +320,25 @@ export class Account extends React.Component<Props, State> {
     this.setState({ showRevokeModal: false })
   }
 
+  // when change pin is clicked, we ensure that they have a pincode and then we navigate to pincode set screen with not verifying, and change pin is true
+
+  goToChangePin = () => {
+    ensurePincode()
+      .then((pinIsCorrect) => {
+        if (pinIsCorrect) {
+          this.props.navigation.navigate(Screens.PincodeSet, {
+            isVerifying: false,
+            changePin: true,
+          })
+        }
+      })
+      .catch((error) => {
+        Logger.error(`${'NavigationService'}@onPress`, 'PIN ensure error', error)
+      })
+
+    // on success, navigate to PincodeSet (with prop for change Pin design)
```

Comment:
nit: This comment is a bit disconnected from the other, maybe put the two together at the top?